spec/models/user_spec.rb

```diff
@@ -20,4 +32,19 @@
       expect(user).to be_valid
     end
   end
+
+  describe 'google login' do
+    it "creates or updates itself from an oauth hash" do
+      user = User.from_omniauth(auth)
+      user.organization_id = organization.id
```

Comment:
???
test does not match the controller, either add additional test to ensure controller is working or extend this one